tests/cases/fourslash/completionListInObjectLiteral5.ts

```diff
@@ -0,0 +1,24 @@
+/// <reference path="fourslash.ts" />
+
+////const o = 'something' 
+////const obj = {
+////    prop: o/*1*/,
```

Comment:
I just merged #41539—if you merge/rebase with master, you could add a marker for a shorthand property assignment.

Reply:
OK~done